src/mmw/js/src/data_catalog/templates/error.html

```diff
@@ -0,0 +1,5 @@
+{% if error %}
+    <div class="no-results-panel">
+        <p> <i class="fa fa-exclamation-triangle"></i> {{ error }} </p>
```

Comment:
Prefer the stacked error used [elsewhere](https://github.com/WikiWatershed/model-my-watershed/blob/develop/src/mmw/js/src/core/templates/message.html)